pkg/envconf/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	dryRun                  bool
 	failFast                bool
 	disableGracefulTeardown bool
+	context                 string
```

Comment:
To avoid confusion with Go context, I would suggest naming this `k8sContext`

Reply:
We have a `kubeconfig`, what do you think about `kubecontext`? I am asking so we have only one way to prefix kubernetes things